geomesa-filter/src/test/scala/org/locationtech/geomesa/filter/function/FastPropertyTest.scala

```diff
@@ -49,7 +49,7 @@ class FastPropertyTest extends Specification {
       val sf = new ScalaSimpleFeature("id", sft)
       sf.setAttributes(Array[AnyRef]("myname", "POINT(45 45)"))
 
-      val filter = ECQL.toFilter("testns:name = 'myname'", FastFilterFactory.factory)
+      val filter = FastFilterFactory.toFilter(sft, "testns:name = 'myname'")
```

Comment:
What happens if someone uses "ECQL.toFilter" with a namespace?

I think that case would match what would happen in GeoServer.